homeassistant/components/soma/__init__.py

```diff
@@ -63,7 +63,14 @@ async def async_setup_entry(hass: HomeAssistantType, entry: ConfigEntry):
 
 async def async_unload_entry(hass: HomeAssistantType, entry: ConfigEntry):
     """Unload a config entry."""
-    return True
+    rtn_value = True
+    for component in SOMA_COMPONENTS:
```

Comment:
How about this `asyncio.gather` approach here to parallelize the unload? I saw this in the default scaffold script used for new integrations and a couple other PR comments, so I assume it is a new best practice.
https://github.com/home-assistant/core/blob/dev/script/scaffold/templates/config_flow_oauth2/integration/__init__.py#L85

Reply:
Done. Thanks for pointing me at that.